lib/modules/manager/gomod/extract.ts

```diff
@@ -34,6 +35,19 @@ function getDep(
   return dep;
 }
 
+function getGoDep(lineNumber: number, goVer: string): PackageDependency {
+  return {
+    managerData: {
+      lineNumber,
+    },
+    depName: 'go',
+    depType: 'golang',
+    currentValue: goVer,
+    datasource: GolangVersionDatasource.id,
```

Comment:
supported datasource needs update in index.ts

Reply:
i dont see any updates needed, please elaborate

Reply:
export const supportedDatasources = [GoDatasource.id, GolangVersionDatasource.id];
? this?

Reply:
yes, here
https://github.com/renovatebot/renovate/blob/5fa2aa2f2c0de8eccba46acfde84934a239c4b4b/lib/modules/manager/gomod/index.ts#L14

Reply:
its done

Reply:
may i ask what is it for? since it worked without it xD